external/docker-integration-tests/pom.xml

```diff
@@ -34,6 +34,13 @@
     <sbt.project.name>docker-integration-tests</sbt.project.name>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>db2</id>
+      <url>https://app.camunda.com/nexus/content/repositories/public/</url>
```

Comment:
https://app.camunda.com/nexus/content/repositories/public/db2

> 404 - Path /db2 not found in group repository "Public Repositories" [id=public].

Any idea what's up with this?